source/blog/2018-11-09-the-ember-times-issue-72.md

```diff
@@ -9,7 +9,7 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>.
-We need **Emberistas at JSConf EU**, so submit a talk to the newly opened CFP! 🗣️ Let's take a look into **more informative Routes** 🤓,
+We need **Emberistas at JSConf EU**, so submit a talk to the newly opened CFP! 🗣️ Let's take a look into **more informative Routes** 🤓, there's an update to the RFC about RFCs!
```

Comment:
😅

Reply:
That RFC is too meta for me 😂 I'll gladly accept more sane phrasings.